<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="login_window">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>-90</y>
      <width>1411</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Downloads/Login Window (1).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: 500;
font-size: 15px;
</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: 500;
font-size: 15px;
</string>
    </property>
    <property name="text">
     <string>Password: </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: transparent;</string>
    </property>
    <property name="text">
     <string>Show password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="employee_login_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: 1000;
font-size: 20px;
background-color: rgb(166, 170, 236);
border: none;
</string>
    </property>
    <property name="text">
     <string>Employee Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="admin_login_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: 1000;
font-size: 20px;
background-color: rgb(170, 51, 45);
 border: none;</string>
    </property>
    <property name="text">
     <string>Admin Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="forgot_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>590</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: 1000;
font-size: 15px;
border: none;</string>
    </property>
    <property name="text">
     <string>Forgot your password?</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>551</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Downloads/Image 19.png</pixmap>
    </property>
   </widget>
   <zorder>login_window</zorder>
   <zorder>username_label</zorder>
   <zorder>password_label</zorder>
   <zorder>checkBox</zorder>
   <zorder>username_input</zorder>
   <zorder>password_input</zorder>
   <zorder>employee_login_btn</zorder>
   <zorder>admin_login_btn</zorder>
   <zorder>forgot_label</zorder>
   <zorder>img_label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
